bounties/npm/angular-redactor/1/README.md

```diff
@@ -0,0 +1,3 @@
+# Overview
+
+`angular-redactor` is an angular directive for the Redactor editor, this package is vulnerable to Cross-site Scripting (XSS) attacks when HTML content mode is used.
```

Comment:
The version on the website is still vulnerable, but is `redactor3`.
I downloaded and tried the `demo` (minimal) but doesn't seem to be vulnerable.

Unfortunately I wasn't able to get a ready-to-go version from the website like in the previous disclosure, but I can confirm the version `3` is still unfixed.